### ISSUE TITLE
High CPU load and chunk updates in empty world

### ISSUE THREAD

**Issue by 456Xander** [turbodiesel4598/NuclearCraft#46]
I'm currently playing a Modpack with 80 Mods and wanted to put NC in it. But as soon as I install it and create a new world I get lower FPS than usual an there are always about 10-20 chunk update although I did'nt move for 10 minutes. It also creates very high CPU load of about 70-80% all the Time, without NC it was at constant 30%.

**Comment by turbodiesel4598** [comment]
Which mods do you have installed? If you didn't actually use anything from NC, the only possibility is a nasty interaction between NC and another mod.

**Comment by 456Xander** [comment]
This are the Mods I have installed
[ModList.txt](https://github.com/turbodiesel4598/NuclearCraft/files/380438/ModList.txt)

**Comment by turbodiesel4598** [comment]
Thanks. Well, I was hoping for Mekanism or Mystcraft, as others have reported issues related to them, but it seems there's something else going on.
Which version of NC do you have installed, and do you get the same lag if only NC is installed?

**Comment by 456Xander** [comment]
I have NC 1.8b installed, and without all the other mods the lag isn't there but there is also the same ammount of chunk updates and the cpu is at 30% instead of 20% without any mods.

**Comment by turbodiesel4598** [comment]
Which version of Java do you have installed?

**Comment by 456Xander** [comment]
Java 8u60

Am 26.07.2016 16:23 schrieb "turbodiesel4598" <notifications@github.com>:

**Comment by turbodiesel4598** [comment]
Hmm - I could actually recreate this with Java 7u71, but since updating to Java 8u101, I have not been able to.
Unfortunately, just like all of these types of bugs, it's probably a certain combo of mods causing the issue, and finding that combo can be incredibly time consuming. It's basically up to you whether you want to try to find which mod is conflicting with NC, or just not use NC.
Hopefully these sort of bugs will not be so prevalent in MC 1.10.

**Comment by 456Xander** [comment]
Ok, I profiled my Modpack one time with and one time without NC. There where some really weird things. First, with NC Garbage Collections took about 500ms each instead of 80ms without it and where more Frequent. Second, with NC there where actually Old Generation GC, without NC the Old Gen had a constant 600MB size. And I found a really weird thing when building a reactor. There where Many Objects in Old Space for the next 15 seconds and GC took 5 seconds to clear up. 
![2016-08-13_12h48_15](https://cloud.githubusercontent.com/assets/16427898/17642840/2c119f24-6156-11e6-9ebc-be528484c6b7.png)
This is a graph of the Memory Usage shortly after I built the Reactor. I also noticed severe Slowdowns when building a Reactor but couldn't reproduce that.
